### PR TITLE
Added `GenereateDocumentationFile` So XML Is Generated and Included in NuGets When Packed

### PR DESCRIPTION
**Pull request by AristurtleDev**
## Description
Currently, the NuGet's are missing the XML documentation file to provide intellisense.  There was originally a PR (#7999) by @skyebird189 to resolve this, but they have chosen to close their PR.  

Following the advice from @harry-cpp in that original PR, I have updated the following files to include the `<GenerateDocumentationFile>true</GenerateDocumentationFile>` statement so the XML is properly generated and packed along with future NuGets:

* https://github.com/MonoGame/MonoGame/blob/develop/MonoGame.Framework/Directory.Build.props
* https://github.com/MonoGame/MonoGame/blob/develop/MonoGame.Framework.Content.Pipeline/MonoGame.Framework.Content.Pipeline.csproj

Reference Issue #8042

### CODE REVIEW

**Review by SimonDarksideJ — CHANGES_REQUESTED** [review]
Can you revert the changes to ```MonoGame.Framework.Content.Pipeline/MonoGame.Framework.Content.Pipeline.csproj``` as they are not within the scope of this PR

**Comment by AristurtleDev** [comment]
@SimonDarksideJ I can revert it but before doing can I ask why it is out of scope?

The `MonoGame.Framework.Content.Pipeline` NuGet is used by several people, myself included, when creating third party extensions for the MGCB Editor, which is also missing the generated XML documentation for intellisense.

**Comment by SimonDarksideJ** [comment]
Because this change is for the Build props update and not for the content pipeline project? (which only adds empty spaces)

**Comment by skyebird189** [comment]
If you look at my original PR #7999, @harry-cpp asked it to be added to both directory.build as well as the content pipeline project.

**Comment by SimonDarksideJ** [comment]
> If you look at my original PR https://github.com/MonoGame/MonoGame/pull/7999, @harry-cpp asked it to be added to both directory.build as well as the content pipeline project.

Ahh, well then you need to only do the change to add the ```<GenerateDocumentationFile>true</GenerateDocumentationFile>``` and not the additional whitespace that was added and all will be good

**Comment by AristurtleDev** [comment]
@SimonDarksideJ Apologies, I think the whitespace was added by VScode autoformatting on save. Currently at a Dr appointment, will fix as soon as im back home later this afternoon.

**Comment by AristurtleDev** [comment]
@SimonDarksideJ Both files have been corrected so that the diff only shows the addition from adding the `GenerateDocumentationFile` element.

**Review by SimonDarksideJ — APPROVED** [review]

**Comment by SimonDarksideJ** [comment]
Great work @AristurtleDev ! now just waiting on a maintainer review :S

**Comment by harry-cpp** [comment]
Thanks @AristurtleDev, merging!

**Pull request merged**
